pydicom/tests/test_uid.py

```diff
@@ -98,8 +27,8 @@ def testGenerateUID(self):
                                '1.2..3.',
                                '1.a.2.',
                                '1.01.1.'):
-            self.assertRaises(ValueError,
-                              lambda: generate_uid(prefix=invalid_prefix))
+            with pytest.raises(ValueError):
+                generate_uid(prefix=invalid_prefix)
 
```

Comment:
Another policy question (not directly related to the code) - shall we use pytest for all tests now? What does that gain, given that standard unit tests can be executed by pytest, but not vice verse?